docs/_static/nbsphinx-gallery.css

```diff
@@ -0,0 +1,31 @@
+.nbsphinx-gallery {
```

Comment:
Maybe add a comment explaining what's going on. Or even better, open a GitHub issue and link to it.

Reply:
I have a comment here: https://github.com/Qiskit-Extensions/qiskit-experiments/blob/2ba6109e11fc086855d8360ad173a04f094dc7d4/docs/conf.py#L80

I can write an issue too.